pages/common/docker.md

```diff
@@ -18,6 +18,10 @@
 
 `docker stop {{container}}`
 
+- Pull a image from docker registery:
```

Comment:
Minor typo in `registry`, and maybe this would read better?
```suggestion
- Pull an image from the docker registry:
```

Reply:
Thank you for your suggestion